src/pluginscheduler.c

```diff
@@ -406,8 +406,6 @@ scheduler_phase_cleanup (plugins_scheduler_t sched, int start, int end)
           plugin->deps = NULL;
           element = element->next;
         }
-      g_slist_free (sched->list[category]);
```

Comment:
I don't have the internals knowledge to judge this one.

Reply:
For reference, relevant Valgrind output:
==31648== 4,248 bytes in 177 blocks are definitely lost in loss record 624 of 683
==31648==    at 0x4C31B25: calloc (in /usr/lib/valgrind/vgpreload_memcheck-amd64-linux.so)
==31648==    by 0x5322A80: g_malloc0 (in /usr/lib/x86_64-linux-gnu/libglib-2.0.so.0.5600.3)
==31648==    by 0x117A0A: plugin_add (pluginscheduler.c:89)
==31648==    by 0x117E97: plugins_scheduler_enable (pluginscheduler.c:206)
==31648==    by 0x11827B: plugins_scheduler_init (pluginscheduler.c:293)
==31648==    by 0x110983: attack_network (attack.c:1043)
==31648==    by 0x11485C: handle_client (openvassd.c:440)
==31648==    by 0x114A17: scanner_thread (openvassd.c:494)
==31648==    by 0x119391: create_process (processes.c:90)
==31648==    by 0x115041: main_loop (openvassd.c:712)
==31648==    by 0x115AF4: main (openvassd.c:1081)